web/template/voucher/one_login_identity_details.gohtml

```diff
@@ -7,7 +7,7 @@
 {{ define "main" }}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-xl">{{ tr .App "yourIdentityConfirmedWithOneLogin" }}</h1>
+            {{ template "notification-banner" (notificationBanner .App "success" (trHtml .App "youHaveSuccessfullyConfirmedYourIdentitySuccess:voucher") "success" "contents" ) }}
```

Comment:
the summary list needs updating here too

Reply:
I'd left this as had assumed we had been requested to not play back DOB and address but if thats not the case I'll update.

Reply:
that is a good question, but I assumed we'd want the same style design even if not all the data